docs/data/joy/components/circular-progress/circular-progress.md

```diff
@@ -93,6 +93,14 @@ You can use these to customize the component with both the `sx` prop and the the
 
 {{"demo": "CircularProgressVariables.js", "hideToolbar": true, "bg": "gradient"}}
 
+## Common examples
+
+### Count up feature
```

Comment:
```suggestion
## 3rd-party integration

### use-count-up
```

@danilo-leal What do you think about grouping 3rd-party libraries in `h2`?

We have similar thing on [input](https://mui.com/joy-ui/react-input/#third-party-formatting) but it is `h3`.

Reply:
I think I'd still leave it in the Common examples, as it would be more consistent with the Input. Maybe the "Third-party integration" title works but it isn't immediately clear when compared to "Third-party formatting" (Input's case). 🤔